packages/v4/patternfly-docs/content/design-guidelines/styles/icons/IconRecommendations.js

```diff
@@ -53,154 +51,142 @@ const iconSvgs = {
+              onClear={() => setSearchValue('')}
+            />
+          </ToolbarItem>
+          <ToolbarItem align={{ default: 'alignRight' }}>
+            <b>{filteredRows.length} items</b>
+          </ToolbarItem>
+        </ToolbarContent>
+      </Toolbar>
+      <Divider />
+      <Table
+        aria-label="Updated icons table"
+        className="ws-icons-recommendations"
+        variant="compact"
+      >
+        <Thead>
+          <Tr>
+            <Th sort={getSortParams(0)}>Old icon</Th>
+            <Th sort={getSortParams(1)}>Updated icon</Th>
+            <Th>Contextual usage</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {sortedRows.map(recommendation => (
+            <Tr key={recommendation[0].iconName}>
+              <Td data-label="Old icon" modifier="fitContent">
+                {recommendation.map(oldRec => {
+                  if (oldRec.iconType === "old") {
+                    const oldIconSvg = oldRec.reactIcon === 'svg' &&
+                      <img src={iconSvgs[oldRec.iconName]} className="ws-icon-svg" alt={`${oldRec.iconName} icon`}/>;
+                    const OldIcon = oldRec.reactIcon !== 'svg' && icons[oldRec.reactIcon];
+                    return (
+                      <div className="ws-recommendations-entry" key={oldRec.iconName + oldRec.iconType + oldRec.iconUsage}>
+                        <span className="ws-recommendations-icon">
+                          {OldIcon && <OldIcon/>}
+                          {oldIconSvg}
+                        </span>
+                          {oldRec.iconName}
+                      </div>
+                    );
+                  } else return '';
+                })}
+              </Td>
+              <Td data-label="Updated icon"  modifier="fitContent">
+                {recommendation.map(newRec => {
+                  if (newRec.iconType === "new") {
+                    const newIconSvg = newRec.reactIcon === 'svg' &&
+                      <img src={iconSvgs[newRec.iconName]} className="ws-icon-svg" alt={`${newRec.iconName} icon`}/>;
+                    const NewIcon = newRec.reactIcon !== 'svg' && icons[newRec.reactIcon];
+                    return (
+                      <div className="ws-recommendations-entry" key={newRec.iconName + newRec.iconType + newRec.iconUsage}>
+                        <span className="ws-recommendations-icon">
+                          {NewIcon && <NewIcon/>}
+                          {newIconSvg}
+                          {newRec.reactIcon === 'PF-Spinner-Component' && <Spinner size='md'/>}
+                        </span>
+                        {newRec.iconName}
+                      </div>
+                    );
+                  } else return '';
+                })}
+              </Td>
+              <Td data-label="Contextual usage">
+                {recommendation.map(rec => {
+                  return rec.iconUsage
```

Comment:
We should only display iconUsage for recommended icons (aka 'new' icons), and add block spacing to avoid results with multiple descriptions from running together (see `pficon-dragdrop` row in PR preview for 3 contextual usages running together)

```suggestion
                  if (rec.iconType === 'new') {
                    return (
                      <div key={`${rec.style}-${rec.iconName}-contextual-usage`}>
                        {rec.iconUsage}
                      </div>
                    )
                  }
```